### PR TITLE
Add symfony4 support

### PR DESCRIPTION
**Pull request by scodazzim6web**

### CODE REVIEW

File: src/DataCollector/RedisDataCollector.php [hunk 1]
```diff
@@ -76,7 +76,7 @@ public function getTotalExecutionTime(): float
             $time += $value['executiontime'];
 
             return $time;
-        });
+        }) ?? 0;
```
Comment (Oliboy50) [inline src/DataCollector/RedisDataCollector.php]: I'd have said:

```suggestion
        }) ?? 0.0;
```

but I'm surprised that `0` also works fine while there is `declare(strict_types=1);` at the top of the file 🤔 

```php
<?php

declare(strict_types=1);

function test(): float
{
    return 0;
}

var_dump(test());
```

produces `float(0)` 🤷‍♂

**Comment by Stax** [comment]
Hello ! Quand es ce que vous avez prévu de merge cette PR ? J'en ai grandement besoin :D

**Comment by Oliboy50** [comment]
@Stax the OP did not work at M6Web anymore... so I think he won't be able to merge it itself

anyway this PR looks good to me, but I didn't tested it personally... 🤷‍♂ 
maybe someone else tried to use it and will be willing to merge it (more than I am)... @GautierLB maybe?

if it's still not merged tomorrow, I'll try to do it

**Comment by Stax** [comment]
@Oliboy50  ok ! Then I will try it myself on an sf4 project and let you know if it's fine !

**Comment by Oliboy50** [comment]
thanks ❤️ 

since it's a fork, you'll have to tell composer to fetch `scodazzim6web/RedisBundle` instead of `M6Web/RedisBundle`

something like that maybe:

```json
{
    "repositories": [
        {
            "type": "vcs",
            "url": "https://github.com/scodazzim6web/RedisBundle"
        }
    ],
    "require": {
        "M6Web/RedisBundle": "dev-add-symfony4-support"
    }
}
```

**Comment by GautierLB** [comment]
> anyway this PR looks good to me, but I didn't tested it personally... 🤷‍♂    
I'm in the same case @Oliboy50 😅

**Comment by Stax** [comment]
Done in https://github.com/M6Web/RedisBundle/pull/53

**Pull request closed without merging**